ci-operator/jobs/openshift/cluster-network-operator/openshift-cluster-network-operator-master-presubmits.yaml

```diff
@@ -1412,6 +1412,89 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-gcp-ovn,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
do you want this running every time? or is it just a job we want to be able to trigger from time to time with the /test commend in github?

Reply:
We can surely make it optional. I'll make the changes.